### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by fatimarabab**
## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [x] My code follows the style guidelines enforced by static analysis tools.
- [x] My changes generate no new warnings.
- [x] New and existing unit tests pass locally with my changes.

## Pull Request Details

Wrike Ticket - [Create a new subsection in deployment to talk about the 'http://local.rosalution.cgds/rosalution/' to access the deployments after run](https://www.wrike.com/open.htm?id=1043807806)
Wrike Ticket - [Add documentation to Main Rosalution README on how to 'Import a new analysis case to Rosalution'](https://www.wrike.com/open.htm?id=1043792932)

Changes made:

- Added a section to detail how to upload a new case onto Rosalution. Added an image cluster demonstrating this.
- Added a section detailing how to access Rosalution when locally deployed.
- Replaced Login images with image cluster showing development and production login direction to Rosalution's home page.

**To Review:**

- [x] Static Analysis by Reviewer
- [x] The changes made in the README are correctly being displayed.
- [x] The detailed notes are accurate and sufficient for the respective sections.
- [x] All Github Actions checks have passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -172,8 +172,18 @@ User permissions according to a type of user is planned to become available in a
...
 |user07 | Bioinformatics-section user
 
-![Login page](./docs/figures/login.png "Login page")
-![User logged in](./docs/figures/logged-in.png "Developer user logged in")
+![Rosalution logins](./docs/figures/rosalution-logins.png "Rosalution logins")
+
+### Adding Cases
+
+Users can create new cases on Rosalution by clicking on the '+' card to add a new case, as shown in the below figures.
+Doing this will prompt a dialog to import files to create a new Rosalution case. These files must be in .json format.
```
Comment (SeriousHorncat) [inline README.md:180]:
```suggestion
Doing so prompts an upload dialog to import files to create a new Rosalution case. These files must be in .json format.
```

File: README.md [hunk 2]
```diff
@@ -172,8 +172,18 @@ User permissions according to a type of user is planned to become available in a
...
+Once uploaded, a 'Successful import' dialog will be displayed.
```
Comment (SeriousHorncat) [inline README.md:181]:
```suggestion
Once uploaded, a 'Successful import' dialog displays.
```

File: README.md [hunk 3]
```diff
@@ -270,6 +280,10 @@ docker-compose -f docker-compose.local-production.yml up --build
  compose file to use, in this case `docker-compose.local-production.yml` and `--build` flag is used to force Docker
  Compose to rebuild the images.
 
+#### Accessing Deployed Rosalution
```
Comment (SeriousHorncat) [inline README.md:283]:
```suggestion
#### Accessing A Locally Deployed Rosalution
```

File: README.md [hunk 4]
```diff
@@ -270,6 +280,10 @@ docker-compose -f docker-compose.local-production.yml up --build
...
+
+To access the Rosalution site after local deployment is run, open the following URL in your browser[http://local.rosalution.cgds/rosalution/](http://local.rosalution.cgds/rosalution/)
```
Comment (SeriousHorncat) [inline README.md:285]:
```suggestion
To access the locally deployed Rosalution application after running `docker compose up`, visit the [http://local.rosalution.cgds/rosalution/](http://local.rosalution.cgds/rosalution/) URL in your browser. 
```

**Review by SeriousHorncat — CHANGES_REQUESTED** [review]
A tip for simplifying language in the documentation.  Try to use present tense and active voice. 
 When a sentence has the word "will." in it, it can be rewritten to be more concise. 

"Once uploaded, a 'Successful import' dialog **will be displayed**." to
"Once uploaded, a 'Successful import' dialog **displays**".

I forget this myself, but I try to update it when I find it.

File: README.md (hunk 2 above)
Comment (fatimarabab) [reply README.md:181]: Thanks for pointing this out! Added suggestion.

**Review by SeriousHorncat — APPROVED** [review]

**Pull request merged**